src/App.test.js

```diff
@@ -1,8 +1,8 @@
-import { render, screen } from '@testing-library/react';
-import App from './App';
+import { render, screen } from '@testing-library/react'
+import App from './App'
 
-test('renders learn react link', () => {
-  render(<App />);
-  const linkElement = screen.getByText(/learn react/i);
-  expect(linkElement).toBeInTheDocument();
-});
+test('renders a playerListItem', () => {
```

Comment:
Since it is a list, maybe check that there are 6 list items? Also later, when you use the `App.js` as a center of your data, it will get pretty hard to test it. Especially with hooks etc. Might be better to not test it as you are writing tests for each component individually 🤔